llvm/test/CodeGen/X86/atomic-non-integer.ll

```diff
@@ -811,8 +854,7 @@ define void @store_bfloat(ptr %fptr, bfloat %v) {
   ret void
 }
```

Comment:
why is this codegen so poor vs store_half?

Reply:
It's ABI promotes to float vs. ABI promotes to i16